File: crates/auto-lang/test/a2r/019_blocks/blocks.expected.rs
fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn multiply(a: i32, b: i32) -> i32 {
    a * b
}

fn main() {
    let sum: i32 = add(5, 3);
    let product: i32 = multiply(4, 7);
    let result: i32 = sum + product;
    println!("Sum: {} {} {} {} {}", sum, "Product:", product, "Result:", result);
}

File: crates/auto-lang/test/a2ts/019_blocks/blocks.expected.ts
function add(a: number, b: number): number {
    a + b;
}

function multiply(a: number, b: number): number {
    a * b;
}

function main(): void {
    const sum = add(5, 3);
    const product = multiply(4, 7);
    const result = sum + product;
    console.log("Sum:", sum, "Product:", product, "Result:", result);
}

main();
